glossary.md

```diff
@@ -1,22 +1,149 @@
+  End-of-Life is usually how this term is used.
+* **ESM**: [ECMAScript Module][] - Standard for JavaScript modules.
+* **ETW**: [Event Tracing for Windows][] - Provides a way to trace events in Windows systems.
+* **FFDC**: First Failure Data Capture - Logs, traces, and dumps produced by default on program error.
+* **FIPS**: [Federal Information Processing Standards][] - Set of standards for use in computer systems by non-military
+  government agencies and government contractors.
+* **FS**: File System.
+* **Godbolt**: [Compiler Explorer][] - Tool for running compilers interactively from a web browser.
+* **HTTP**: [Hypertext Transfer Protocol][] - An application protocol for distributed, collaborative, hypermedia
+  information systems. It is the foundation of data communication on the World Wide Web.
+* **ICU**: [International Components for Unicode][] - Library providing support for Unicode.
+* **IDE**: [Integrated Development Environment][] - A software application that provides comprehensive facilities to
+  computer programmers for software development.
+* **IETF**: [Internet Engineering Task Force][] - An international community responsible for developing and promoting
+  Internet standards.
+* **IIRC**: If I Recall Correctly.
+* **IIUC**: If I Understand Correctly.
+* **IMHO**: In My Humble/Honest Opinion.
+* **IMO**: In My Opinion.
+* **IPC**: [Inter-Process Communication][] - Mechanism allowing processes to communicate with each other.
+* **JIT**: [Just In Time][] - Method of executing computer code during runtime.
+* **JS**: [JavaScript][] - A high-level, interpreted programming language that is primarily used to create dynamic and
+  interactive content on web pages. It is commonly used for client-side scripting in web development.
+* **JS/C++ boundary**: Boundary between V8's runtime and JS code execution, often crossed when calling JS functions
+  with C++ linkage.
+* **JSON**: [JavaScript Object Notation][] - A lightweight data interchange format that is easy for humans to read and
+  write, and easy for machines to parse and generate. It is commonly used for transmitting data between a server and a
+  web application.
+* **LE**: Little [Endian][] - A Byte Order where the smallest bit comes first. The opposite of **BE**.
+* **LGTM/SGTM**: Looks/Sounds good to me - Commonly used to approve a code review.
+* **LTS**: [Long Term Support][] - Support provided for a software version for an extended period.
+* **MDN**: [Mozilla Development Network][] - Resource for web developers.
+* **MVC**: [Model-View-Controller][] - A software design pattern commonly used for developing user interfaces. It
+  separates the application into three interconnected components: the model (data), the view (presentation), and the
+  controller (logic).
+* **Native modules/addons**: Modules implemented in C/C++ that expose interfaces callable from JS.
```

Comment:
It doesn't have to be C nor C++ AFAIK
```suggestion
* **Native modules/addons**: Modules implemented in e.g. C or C++ that expose interfaces callable from JS.
```

Reply:
This comment was marked as resolved but not addressed, if you disagree with the suggestion, please leave a comment explaining.

Reply:
Apologies if I mis-resolved it. I'll make all the changes later today